src/pages/index.js

```diff
@@ -2,34 +2,116 @@ import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import "../pages/index.css";
 import {
-  initialCards,
   cardContainer,
   cardSelector,
   cardForm,
   settings,
   buttonEdit,
-  titleInput,
   buttonAdd,
-  linkInput,
   nameInput,
   jobInput,
+  changeAvatarbtn,
+  apiOptions,
+  avatarInput,
+  savingBtn,
 } from "../utils/constants.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
```

Comment:
Please, delete `this._form.reset();` from line 35 in `PopupWithForm.js`

You need to **reset** the form only  in a `then` block in order not to do that immediately after `submit` because when there is an error, the modal will still be opened and the user can try to send the data again to the server without having to type the data into the inputs once more.